configs/test/bot/setup/android.bash

```diff
@@ -13,6 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function run_bot () {
+  bot_directory=$INSTALL_DIRECTORY/bots/$(echo $1 | sed s/:/-/)
+  mkdir -p $bot_directory
+  cp -r $INSTALL_DIRECTORY/clusterfuzz $bot_directory/clusterfuzz
```

Comment:
what if dir already existed, can we delete and recreate

Reply:
Done.